server/src/models/hospital.js

```diff
@@ -0,0 +1,33 @@
+const mongoose = require("mongoose");
```

Comment:
Since this is a separate issue please create a different PR for it.
I can award points on the basis of merged PR.